### DISCUSSION TITLE
Limited public beta link 🎉

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [community/community#134324] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hey Devs, just dropping by with the link we all want 😄 

"We believe every developer can be an AI engineer with the right tools and training. From playground to coding with the model in Codespaces to production deployment via Azure, GitHub Models shows you how simple it can be. Sign up for the limited public beta [HERE](https://gh.io/models)."

Want to learn more about GitHub Models? Here is the [full blogpost announcement](https://github.blog/news-insights/product-news/introducing-github-models/).

**Comment by yuyc2022** [comment]
good!
